projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -124,16 +124,16 @@ public static function init() {
 
 		add_action( 'admin_menu', array( 'Jetpack_Custom_CSS', 'menu' ) );
 
-		if ( isset( $_POST['safecss'] ) && false === strstr( $_SERVER['REQUEST_URI'], 'options.php' ) ) {
+		if ( isset( $_POST['safecss'] ) && isset( $_SERVER['REQUEST_URI'] ) && false === strstr( filter_var( wp_unslash( $_SERVER['REQUEST_URI'] ) ), 'options.php' ) ) {
```

Comment:
To preserve the existing logic, this should rather be
```suggestion
		if ( isset( $_POST['safecss'] ) && ( ! isset( $_SERVER['REQUEST_URI'] ) || false === strstr( filter_var( wp_unslash( $_SERVER['REQUEST_URI'] ) ), 'options.php' ) ) ) {
```
or
```suggestion
		if ( isset( $_POST['safecss'] ) && ! ( isset( $_SERVER['REQUEST_URI'] ) && false !== strstr( filter_var( wp_unslash( $_SERVER['REQUEST_URI'] ) ), 'options.php' ) ) ) {
```